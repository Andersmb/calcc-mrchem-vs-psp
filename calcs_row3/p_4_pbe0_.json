{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -0.11505923936168783
          ],
          "radius": 1.23
        },
        {
          "center": [
            0.0,
            1.93300071,
            1.7678637406383122
          ],
          "radius": 0.79
        },
        {
          "center": [
            0.0,
            -1.93300071,
            1.7678637406383122
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "p",
          "xyz": [
            0.0,
            0.0,
            -0.11505923936168783
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            1.93300071,
            1.7678637406383122
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            -1.93300071,
            1.7678637406383122
          ]
        }
      ],
      "multiplicity": 2
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "p_4_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        2.440423242214155e-19,
        -1.7236413942820428e-18
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.22593390539909042,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            4.2935272381827876e-06,
            1.2504735195110365e-07,
            0.22593390535825986
          ],
          "vector_el": [
            4.293432468318949e-06,
            1.2496380811258981e-07,
            -1.5839049854187341
          ],
          "vector_nuc": [
            9.476986383850587e-11,
            8.354383851383318e-11,
            1.809838890776994
          ]
        }
      },
      "geometry": [
        {
          "symbol": "P",
          "xyz": [
            0.0,
            0.0,
            -0.11505923936168783
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            1.93300071,
            1.7678637406383122
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            -1.93300071,
            1.7678637406383122
          ]
        }
      ],
      "multiplicity": 2,
      "orbital_energies": {
        "energy": [
          -77.22385789764527,
          -6.622955464885197,
          -4.764597633415643,
          -4.75664900373061,
          -4.753862783443579,
          -0.6556584288270182,
          -0.3993727113942266,
          -0.32371353412333187,
          -0.2665086815986349,
          -77.21929221609638,
          -6.6181296051969065,
          -4.753789936928411,
          -4.751356916356135,
          -4.7480074687301554,
          -0.6208325582563133,
          -0.39300708456091943,
          -0.3090201224093549
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -199.1806120475981
      },
      "scf_energy": {
        "E_ee": 165.66205921484394,
        "E_eext": 0.0,
        "E_el": -353.76900798413226,
        "E_en": -837.4922367702128,
        "E_kin": 341.8613093784111,
        "E_next": 0.0,
        "E_nn": 11.375961395679775,
        "E_nuc": 11.375961395679775,
        "E_tot": -342.3930465884525,
        "E_x": -5.822686490713643,
        "E_xc": -17.97745331646079,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 165.5946081303652,
        "E_eext": 0.0,
        "E_el": -351.98302556467405,
        "E_en": -833.3624517017154,
        "E_kin": 339.5297528380138,
        "E_next": 0.0,
        "E_nn": 11.375961395679775,
        "E_nuc": 11.375961395679775,
        "E_tot": -340.60706416899427,
        "E_x": -5.810178374227946,
        "E_xc": -17.93475645710972,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 165.19294107571116,
              "E_eext": 0.0,
              "E_el": -353.70451327707946,
              "E_en": -833.8040526819351,
              "E_kin": 338.6284450849544,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.3285518813997,
              "E_x": -5.804488515115484,
              "E_xc": -17.917358240694455,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.3285518813997,
            "energy_update": 1.7214877124054055,
            "mo_residual": 0.16955527933667805,
            "wall_time": 91.507559272
          },
          {
            "energy_terms": {
              "E_ee": 165.44798403305344,
              "E_eext": 0.0,
              "E_el": -353.75738442237054,
              "E_en": -835.8262733115898,
              "E_kin": 340.38160552327133,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.38142302669075,
              "E_x": -5.813198256735911,
              "E_xc": -17.947502410369587,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.38142302669075,
            "energy_update": 0.05287114529107839,
            "mo_residual": 0.05200549111596418,
            "wall_time": 63.824924787
          },
          {
            "energy_terms": {
              "E_ee": 165.53313864689798,
              "E_eext": 0.0,
              "E_el": -353.7655618242485,
              "E_en": -836.7089956832256,
              "E_kin": 341.18849584023604,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.3896004285687,
              "E_x": -5.8171711575239895,
              "E_xc": -17.96102947063293,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.3896004285687,
            "energy_update": 0.008177401877958346,
            "mo_residual": 0.02506808829592295,
            "wall_time": 57.102585809
          },
          {
            "energy_terms": {
              "E_ee": 165.65948470959844,
              "E_eext": 0.0,
              "E_el": -353.76828392360795,
              "E_en": -837.5090968793661,
              "E_kin": 341.87936679980555,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.3923225279282,
              "E_x": -5.821883545657606,
              "E_xc": -17.976155007988247,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.3923225279282,
            "energy_update": 0.002722099359459662,
            "mo_residual": 0.026467776878076565,
            "wall_time": 56.399987168
          },
          {
            "energy_terms": {
              "E_ee": 165.68283949143586,
              "E_eext": 0.0,
              "E_el": -353.7688252629539,
              "E_en": -837.689867280267,
              "E_kin": 342.04106705562043,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.3928638672741,
              "E_x": -5.823105568131054,
              "E_xc": -17.979758961612188,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.3928638672741,
            "energy_update": 0.0005413393459434701,
            "mo_residual": 0.012853349461473,
            "wall_time": 58.063539421
          },
          {
            "energy_terms": {
              "E_ee": 165.66977334218893,
              "E_eext": 0.0,
              "E_el": -353.76897516940505,
              "E_en": -837.5446906737856,
              "E_kin": 341.9068522975599,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.3930137737253,
              "E_x": -5.822776513742265,
              "E_xc": -17.978133621626043,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.3930137737253,
            "energy_update": 0.00014990645115631196,
            "mo_residual": 0.007373319066265727,
            "wall_time": 56.707592953
          },
          {
            "energy_terms": {
              "E_ee": 165.6663840890325,
              "E_eext": 0.0,
              "E_el": -353.769000104988,
              "E_en": -837.489681129929,
              "E_kin": 341.8546010856298,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.3930387093082,
              "E_x": -5.822712142400895,
              "E_xc": -17.977592007320418,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.3930387093082,
            "energy_update": 2.493558292826492e-05,
            "mo_residual": 0.0025802679473052644,
            "wall_time": 59.638319058
          },
          {
            "energy_terms": {
              "E_ee": 165.6617267370999,
              "E_eext": 0.0,
              "E_el": -353.76900605510167,
              "E_en": -837.4797978190844,
              "E_kin": 341.8489459523303,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.3930446594219,
              "E_x": -5.822621063982664,
              "E_xc": -17.97725986146485,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.3930446594219,
            "energy_update": 5.950113688868441e-06,
            "mo_residual": 0.0013264533999598756,
            "wall_time": 59.395607243
          },
          {
            "energy_terms": {
              "E_ee": 165.66027658675736,
              "E_eext": 0.0,
              "E_el": -353.7690072606614,
              "E_en": -837.4794781493003,
              "E_kin": 341.850018097817,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.3930458649816,
              "E_x": -5.822613354875834,
              "E_xc": -17.977210441059523,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.3930458649816,
            "energy_update": 1.2055597267135454e-06,
            "mo_residual": 0.0006361342423789281,
            "wall_time": 59.004789961
          },
          {
            "energy_terms": {
              "E_ee": 165.66118284423803,
              "E_eext": 0.0,
              "E_el": -353.7690078170956,
              "E_en": -837.48988165076,
              "E_kin": 341.8597505333884,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.3930464214158,
              "E_x": -5.822671253201821,
              "E_xc": -17.97738829076018,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.3930464214158,
            "energy_update": 5.564342018260504e-07,
            "mo_residual": 0.0005126425674533399,
            "wall_time": 60.900532087
          },
          {
            "energy_terms": {
              "E_ee": 165.6618122580111,
              "E_eext": 0.0,
              "E_el": -353.7690079507792,
              "E_en": -837.4935512585755,
              "E_kin": 341.86287780516926,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.39304655509943,
              "E_x": -5.822689489808075,
              "E_xc": -17.977457265576035,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.39304655509943,
            "energy_update": 1.336836135124031e-07,
            "mo_residual": 0.00022719826524274646,
            "wall_time": 60.400798804
          },
          {
            "energy_terms": {
              "E_ee": 165.66205921484394,
              "E_eext": 0.0,
              "E_el": -353.76900798413226,
              "E_en": -837.4922367702128,
              "E_kin": 341.8613093784111,
              "E_next": 0.0,
              "E_nn": 11.375961395679775,
              "E_nuc": 11.375961395679775,
              "E_tot": -342.3930465884525,
              "E_x": -5.822686490713643,
              "E_xc": -17.97745331646079,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.3930465884525,
            "energy_update": 3.3353046546835685e-08,
            "mo_residual": 9.490746857828761e-05,
            "wall_time": 72.23923272
          }
        ],
        "wall_time": 755.228149838
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
