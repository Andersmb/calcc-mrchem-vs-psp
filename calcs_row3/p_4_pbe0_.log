Starting job 5688405 on c1-[24,36,38],c2-49,c3-18,c5-49 at Fri May 20 16:58:24 CEST 2022

Copying p_4_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying p_4_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688405       p_4_pbe0_        180                                             00:14:14      0:0 
5688405.bat+      batch         36        1   00:00:02          0   00:00:02   00:14:14      0:0 
5688405.ext+     extern        180        6   00:00:00          0   00:00:00   00:14:14      0:0 
5688405.0      mrchem.x        180       30   00:13:37         29   00:30:18   00:13:46      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688405                                                                          
5688405.bat+     19332K          0     19332K        0              0          0 
5688405.ext+          0          0          0        0              0          0 
5688405.0      4345976K         29 2767728776        0             11          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688405                                                                                               
5688405.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688405.ext+        0.00M               0          0.00M            0                0              0 
5688405.0           6.77M               0          6.74M       63.09M               14         31.87M 

Job 5688405 completed at Fri May 20 17:12:37 CEST 2022
