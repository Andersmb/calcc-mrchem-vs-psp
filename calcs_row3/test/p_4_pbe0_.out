                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        0.000000       -0.115059
    1       H            :        0.000000        1.933001        1.767864
    2       H            :        0.000000       -1.933001        1.767864
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               9
 Beta electrons          :                                               8
 Total electrons         :                                              17
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999884814e-01
    1    1    a          :                              9.999999266620e-01
    2    1    a          :                              1.000000124492e+00
    3    1    a          :                              1.000000123279e+00
    4    1    a          :                              1.000000000000e+00
    5    1    a          :                              9.999994569784e-01
    6    1    a          :                              1.000001134541e+00
    7    1    a          :                              1.000001200193e+00
    8    1    a          :                              1.000000000000e+00
    9    1    b          :                              9.999999884814e-01
   10    1    b          :                              9.999999266620e-01
   11    1    b          :                              1.000000124492e+00
   12    1    b          :                              1.000000123279e+00
   13    1    b          :                              1.000000000000e+00
   14    1    b          :                              9.999994569784e-01
   15    1    b          :                              1.000001134541e+00
   16    1    b          :                              1.000001198751e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                339.529752838014
 E-N energy              :            (au)               -833.362451701715
 Coulomb energy          :            (au)                165.594608130365
 Exchange energy         :            (au)                 -5.810178374228
 X-C energy              :            (au)                -17.934756457110
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 11.375961395680
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -351.983025564674
 Nuclear energy          :            (au)                 11.375961395680
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.406070641690e+02
                         :      (kcal/mol)             -2.137341579743e+05
                         :        (kJ/mol)             -8.942637175450e+05
                         :            (eV)             -9.268389973853e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -76.659047654737
    1    1    a          :            (au)                 -6.551277840463
    2    1    a          :            (au)                 -4.693573223757
    3    1    a          :            (au)                 -4.686695253510
    4    1    a          :            (au)                 -4.682866485768
    5    1    a          :            (au)                 -0.662600576672
    6    1    a          :            (au)                 -0.404320650196
    7    1    a          :            (au)                 -0.300038843172
    8    1    a          :            (au)                 -0.250410809195
    9    1    b          :            (au)                -76.654805225215
   10    1    b          :            (au)                 -6.546500300525
   11    1    b          :            (au)                 -4.684249679345
   12    1    b          :            (au)                 -4.680432261480
   13    1    b          :            (au)                 -4.677161827264
   14    1    b          :            (au)                 -0.629636402399
   15    1    b          :            (au)                 -0.395195686063
   16    1    b          :            (au)                 -0.279553702416
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -197.438366422177
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.123106e+00        -340.607064168994         -3.406071e+02
    1          1.695553e-01        -342.328551881400         -1.721488e+00
    2          5.200549e-02        -342.381423026691         -5.287115e-02
    3          2.506809e-02        -342.389600428569         -8.177402e-03
    4          2.646778e-02        -342.392322527928         -2.722099e-03
    5          1.285335e-02        -342.392863867274         -5.413393e-04
    6          7.373319e-03        -342.393013773725         -1.499065e-04
    7          2.580268e-03        -342.393038709308         -2.493558e-05
    8          1.326453e-03        -342.393044659422         -5.950114e-06
    9          6.361342e-04        -342.393045864982         -1.205560e-06
   10          5.126426e-04        -342.393046421416         -5.564342e-07
   11          2.271983e-04        -342.393046555099         -1.336836e-07
   12          9.490747e-05        -342.393046588452         -3.335305e-08
---------------------------------------------------------------------------
                      SCF converged in 12 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        0.000000       -0.115059
    1       H            :        0.000000        1.933001        1.767864
    2       H            :        0.000000       -1.933001        1.767864
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                341.861309378411
 E-N energy              :            (au)               -837.492236770213
 Coulomb energy          :            (au)                165.662059214844
 Exchange energy         :            (au)                 -5.822686490714
 X-C energy              :            (au)                -17.977453316461
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 11.375961395680
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -353.769007984132
 Nuclear energy          :            (au)                 11.375961395680
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.423930465885e+02
                         :      (kcal/mol)             -2.148548788540e+05
                         :        (kJ/mol)             -8.989528137086e+05
                         :            (eV)             -9.316989029161e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.223857897645
    1    1    a          :            (au)                 -6.622955464885
    2    1    a          :            (au)                 -4.764597633416
    3    1    a          :            (au)                 -4.756649003731
    4    1    a          :            (au)                 -4.753862783444
    5    1    a          :            (au)                 -0.655658428827
    6    1    a          :            (au)                 -0.399372711394
    7    1    a          :            (au)                 -0.323713534123
    8    1    a          :            (au)                 -0.266508681599
    9    1    b          :            (au)                -77.219292216096
   10    1    b          :            (au)                 -6.618129605197
   11    1    b          :            (au)                 -4.753789936928
   12    1    b          :            (au)                 -4.751356916356
   13    1    b          :            (au)                 -4.748007468730
   14    1    b          :            (au)                 -0.620832558256
   15    1    b          :            (au)                 -0.393007084561
   16    1    b          :            (au)                 -0.309020122409
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -199.180612047598
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000004        0.000000       -1.583905
 Magnitude               :            (au)                        1.583905
                         :         (Debye)                        4.025885
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.809839
 Magnitude               :            (au)                        1.809839
                         :         (Debye)                        4.600151
---------------------------------------------------------------------------
 Total vector            :        0.000004        0.000000        0.225934
 Magnitude               :            (au)                        0.225934
                         :         (Debye)                        0.574267
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 13m 37s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
